src/pyosmeta/parse_issues.py

```diff
@@ -144,14 +184,20 @@ def get_repo_meta(self, url: str, stats_list: list) -> dict:
         # Small script to get the url (normally the docs) and description of a repo!
         response = requests.get(url)
 
+        # TODO: should this be some sort of try/except how do i catch these
```

Comment:
Also here: i am dealing with various response codes that i want to catch. so i suppose this should be a try/ except but i'm not exactly sure how to implement it here...